modules/core/src/test/scala/gem/Arbitraries.scala

```diff
@@ -32,6 +32,9 @@ trait Arbitraries extends gem.config.Arbitraries  {
       )
     }
 
+  implicit val cogLocation: Cogen[Location] =
```

Comment:
> Do we actually make use of this anywhere yet?

Yes, you need it to test the congruence law for `Eq` … `a == b` implies `f(a) == f(b)` so you need to generate arbitrary `f: A => A`. Defined [here](https://github.com/typelevel/cats/blob/6fefa30bfb85e90d193b53300b8313c4dd6c0201/kernel-laws/src/main/scala/cats/kernel/laws/OrderLaws.scala#L36-L38).